toolchain/check/handle_pattern_list.cpp

```diff
@@ -44,7 +44,17 @@ auto HandleParseNode(Context& context, Parse::TuplePatternId node_id) -> bool {
   // make use of it.
   auto refs_id = context.param_and_arg_refs_stack().EndAndPop(
       Parse::NodeKind::TuplePatternStart);
-  context.node_stack().Push(node_id, refs_id);
+  // TODO: do this at full-pattern level.
+  std::vector<SemIR::InstId> inner_param_insts;
```

Comment:
```suggestion
  llvm::SmallVector<SemIR::InstId> inner_param_insts;
```

We're generally using LLVM types where they exist.